### PR TITLE
Increase default limit on GetPopulationTypes response

### PR DESCRIPTION
**Pull request by thomasridd**
### What

Increase default limit on GetPopulationTypes response

### How to review

Sense check

### Who can review

!me

### CODE REVIEW

File: population/population_types.go [hunk 1]
```diff
@@ -42,7 +44,12 @@ func (c *Client) GetPopulationTypes(ctx context.Context, input GetPopulationType
 		"method": http.MethodGet,
 	}
 
-	urlPath := "population-types"
+	limit := 1000
```
Comment (j-s-rawat) [inline population/population_types.go:47]: Do we want it to be an enormously high no. for future proofing

Comment (thomasridd) [reply population/population_types.go]: I've defaulted to passing no parameters but I suspect we are now having to pass in the number 1000 in every place we use the method

Comment (thomasridd) [reply population/population_types.go]: or perhaps 10,000 as you suggest - though I think if we get over 100 our servers will burst into fire

File: population/population_types.go [hunk 2]
```diff
@@ -43,6 +45,12 @@ func (c *Client) GetPopulationTypes(ctx context.Context, input GetPopulationType
 	}
 
 	urlPath := "population-types"
+	if input.Limit > 0 {
```
Comment (j-s-rawat) [inline population/population_types.go:48]: don't think we need these tests, 'limit' and 'offset' will be initialised to 0 when the structure is passed, and currently if 0 is passed as limit code defaults it to '20' So simply adding what ever limit and offset that is passed in pagination should be ok

**Review by j-s-rawat — APPROVED** [review]
LGTM

**Pull request merged**